Mathlib/Data/Polynomial/Laurent.lean

```diff
@@ -156,6 +161,9 @@ theorem single_eq_C (r : R) : Finsupp.single 0 r = C r := rfl
 set_option linter.uppercaseLean3 false in
 #align laurent_polynomial.single_eq_C LaurentPolynomial.single_eq_C
 
+@[simp] lemma C_apply {t : R} {n : ℤ} : C t n = if n = 0 then t else 0 := by
+  rw [← single_eq_C, Finsupp.single_apply]; aesop
```

Comment:
I don't think this should be a `simp` lemma; [`polynomial.coeff_C`](https://leanprover-community.github.io/mathlib_docs/data/polynomial/basic.html#polynomial.coeff_C) is analogous and also not `simp`.

Reply:
Oh nevermind, I guess this is consistent with `MvPolynomial`.

Reply:
```suggestion
@[simp] lemma C_apply {t : R} {n : ℤ} : C t n = if n = 0 then t else 0 := by
  rw [← single_eq_C, Finsupp.single_apply]; aesop
```
```suggestion
@[simp] lemma C_apply (t : R) (n : ℤ) : C t n = if n = 0 then t else 0 := by
  rw [← single_eq_C, Finsupp.single_apply]; aesop
```